### PR TITLE
Update SharePoint_Downloads_byNewIP.yaml

### PR DESCRIPTION
**Pull request by praveenthepro**
Required items, please complete
   
   Change(s):
   - See guidance below

   Reason for Change(s): Current query is generating too many FP's, so added additional logics to minimize the FP's
   - See guidance below

   Version Updated:
   - Required only for Detections/Analytic Rule templates
   - See guidance below

   Testing Completed:
   - See guidance below

   Checked that the validations are passing and have addressed any issues that are present:
   - See guidance below

# Guidance <- remove section before submitting
-----------------------------------------------------------------------------------------------------------
## Before submitting this PR please ensure that you have read the following sections and filled out the changes, reason for change and testing complete sections:

Thank you for your contribution to the Microsoft Sentinel Github repo.

> Details of the code changes in your submitted PR.  Providing descriptions for pull requests ensures there is context to changes being made and greatly enhances the code review process.  Providing associated Issues that this resolves also easily connects the reason.
   
   Change(s):
   - Updated syntax for XYZ.yaml

   Reason for Change(s):
   - New schema used for XYZ.yaml
   - Resolves ISSUE #1234

   Version updated:
   - Yes
   - Detections/Analytic Rule templates are required to have the version updated

> The code should have been tested in a Microsoft Sentinel environment that does not have any custom parsers, functions or tables, so that you validate no incorrect syntax and execution functions properly.  If your submission requires a custom parser or function, it must be submitted with the PR.

   Testing Completed:
   - Yes

_Note: If updating a detection, you must update the version field._

> Before the submission has been made, please look at running the KQL and Yaml Validation Checks locally.
> https://github.com/Azure/Azure-Sentinel#run-kql-validation-locally

   Checked that the validations are passing and have addressed any issues that are present:
   - Yes/No/Need Help
   
   _Note: Let us know if you have tried fixing the validation error and need help._

> References: 
> - [Guidance for Detection checks](https://github.com/Azure/Azure-Sentinel#pull-request-detection-template-structure-validation-check)
> - [General contribution guidance](https://github.com/Azure/Azure-Sentinel/wiki#what-can-you-contribute-and-how-can-you-create-contributions)
> - [PR validation troubleshooting](https://github.com/Azure/Azure-Sentinel#pull-request)

-----------------------------------------------------------------------------------------------------------

### CODE REVIEW

File: Solutions/Microsoft 365/Analytic Rules/SharePoint_Downloads_byNewIP.yaml [hunk 1]
```diff
@@ -18,35 +17,41 @@ tactics:
...
+  | extend GeoLocation = geo_info_from_ip_address(ClientIP)
+  | summarize StartTimeUtc = min(TimeGenerated), EndTimeUtc = max(TimeGenerated), RecentCount = count() by UserId, UserType, Operation, tostring(GeoLocation), Site_Url, ClientIP, OfficeObjectId, OfficeWorkload, UserAgent;
+  // Join the baseline and recent activity, and calculate the deviation
+  let UserBehaviorAnalysis = userBaseline | join kind=inner (recentUserActivity) on UserId, Operation, GeoLocation, Site_Url, ClientIP
+  | extend Deviation = abs(RecentCount - AvgCount) / AvgCount;
+  // Filter for significant deviations
+  UserBehaviorAnalysis
+  | where Deviation > threshold
+  | project StartTimeUtc, EndTimeUtc, UserId, UserType, Operation, ClientIP, tostring(GeoLocation), Site_Url, OfficeObjectId, OfficeWorkload, UserAgent, Deviation, Count=RecentCount
   | extend timestamp = StartTimeUtc, AccountCustomEntity = UserId, IPCustomEntity = ClientIP, URLCustomEntity = Site_Url
```
Comment (petebryan) [inline Solutions/Microsoft 365/Analytic Rules/SharePoint_Downloads_byNewIP.yaml:53]: We don't need to have these *CustomEntity columns now - I know this wasn't the focus of your work here but whilst you are updating can you please remove these columns and update the entity mappings to just map to the underlying columns.

**Review by petebryan — APPROVED** [review]

**Comment by v-atulyadav** [comment]
Hi @praveenthepro, please look into below KQL validation error. Thanks

image

**Comment by praveenthepro** [comment]
@v-atulyadav , Fixed the KQL error, Kindly please checkout

**Comment by v-atulyadav** [comment]
Thanks @praveenthepro.

**Review by petebryan — APPROVED** [review]

**Pull request merged**